api-reference/_hidden/GridBaseColumn/sortOrder.md

```diff
@@ -10,12 +10,14 @@ firedEvents: optionChanged
 Specifies the sort order of column values.
 
 ---
-By default, rows are sorted according to the data source. Set the **sortOrder** property to sort rows in a required order. If you need to sort by multiple columns, specify the [sortIndex](/api-reference/_hidden/GridBaseColumn/sortIndex.md '{basewidgetpath}/Configuration/columns/#sortIndex') property as well, or otherwise, each sorted column will get a sort index according to the position in the **columns** array.
+By default, rows are sorted according to the data source. To sort rows in an ascending or descending order, set the **sortOrder** property to *"asc"* or *"desc"*. If you need to sort by multiple columns, specify the [sortIndex](/api-reference/_hidden/GridBaseColumn/sortIndex.md '{basewidgetpath}/Configuration/columns/#sortIndex') property as well, or otherwise, each sorted column will get a sort index according to its position in the **columns** array.
```

Comment:
```suggestion
By default, rows are sorted according to the data source. To sort rows in an ascending or descending order, set the **sortOrder** property to *"asc"* or *"desc"*. If you need to sort by multiple columns, specify the [sortIndex](/api-reference/_hidden/GridBaseColumn/sortIndex.md '{basewidgetpath}/Configuration/columns/#sortIndex') property too, otherwise, each sorted column will get a sort index according to its position in the **columns** array.
```

Reply:
When you say specify a property - you don't say specify as what? I am confused. Do you mean select a property?

Reply:
> Do you mean select a property?

No, I mean to set a property to a value. SInce the sortIndex property is self-exaplanatory, can the last sentence be changed to "If you need to sort by multiple columns, specify the [sortIndex](). Otherwise, each sorted column will get a sort index according to its position in the **columns** array."?

Reply:
yes that's fine